comp/logs/agent/agentimpl/serverless.go

```diff
@@ -14,14 +14,19 @@ import (
 	tagger "github.com/DataDog/datadog-agent/comp/core/tagger/def"
 	"github.com/DataDog/datadog-agent/comp/logs/agent"
 	flareController "github.com/DataDog/datadog-agent/comp/logs/agent/flare"
+	compression "github.com/DataDog/datadog-agent/comp/serializer/compression/def"
 	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
 	"github.com/DataDog/datadog-agent/pkg/logs/service"
 	"github.com/DataDog/datadog-agent/pkg/logs/sources"
 	"github.com/DataDog/datadog-agent/pkg/logs/tailers"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 // NewServerlessLogsAgent creates a new instance of the logs agent for serverless
-func NewServerlessLogsAgent(tagger tagger.Component) agent.ServerlessLogsAgent {
+func NewServerlessLogsAgent(tagger tagger.Component, compression compression.Component) agent.ServerlessLogsAgent {
+
+	log.Debugf("ZORK Setting up agent")
```

Comment:
I'm assuming we want to delete this?

Reply:
Yes! That was me trying to debug the serverless integration tests. I swear I had grepped for these...